public/apps/configuration/panels/user-list.tsx

```diff
@@ -0,0 +1,199 @@
+    render: dictView(),
+    truncateText: true,
+  },
+];
+
+export function UserList(props: AppDependencies) {
+  const [userData, setUserData] = useState<InternalUsersListing[]>([]);
+  const [errorFlag, setErrorFlag] = useState(false);
+  const [selection, setSelection] = useState<InternalUsersListing[]>([]);
+  const [isActionsPopoverOpen, setActionsPopoverOpen] = useState(false);
+  const [currentUsername, setCurrentUsername] = useState('');
+  // TODO: fetch current username
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setUserData(await getUserList(props.coreStart.http));
+      } catch (e) {
+        console.log(e);
+        setErrorFlag(true);
+      }
+    };
+
+    fetchData();
+  }, [props.coreStart.http]);
+
+  const handleDelete = async () => {
+    const usersToDelete: string[] = selection.map((r) => r.username);
+    try {
+      await requestDeleteUsers(props.coreStart.http, usersToDelete);
+      // Refresh from server (calling fetchData) does not work here, the server still return the roles
+      // that had been just deleted, probably because ES takes some time to sync to all nodes.
+      // So here remove the selected roles from local memory directly.
+      setUserData(difference(userData, selection));
+      setSelection([]);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setActionsPopoverOpen(false);
+    }
+  };
+
+  const actionsMenuItems = [
+    <EuiContextMenuItem
+      key="edit"
+      onClick={() => {}} // TODO: Redirect to edit page
+      disabled={selection.length !== 1}
+    >
+      Edit
+    </EuiContextMenuItem>,
+    // TODO: Redirect to duplicate page
+    <EuiContextMenuItem key="duplicate" onClick={() => {}} disabled={selection.length !== 1}>
+      Duplicate
+    </EuiContextMenuItem>,
+    <EuiContextMenuItem
+      key="delete"
+      onClick={handleDelete}
+      disabled={selection.length === 0 || selection.some((e) => e.username === currentUsername)}
+    >
+      Delete
+    </EuiContextMenuItem>,
+  ];
+
+  const actionsButton = (
+    <EuiButton
+      iconType="arrowDown"
+      iconSide="right"
+      onClick={() => {
+        setActionsPopoverOpen(true);
+      }}
+    >
+      Actions
+    </EuiButton>
+  );
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiTitle size="l">
+          <h1>Internal users</h1>
+        </EuiTitle>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle size="s">
+              <h3>Internal users ({userData.length})</h3>
+            </EuiTitle>
+            <EuiText size="xs" color="subdued">
+              The Security plugin includes an internal user database. Use this database in place of
+              or in addition to an external authentication system such as LDAP or Active Directory.
+              You can map an internal user to a role from Roles. First, click into the detail page
```

Comment:
"Roles" looks like a link from the mockup

Reply:
Good catch. Added an empty link.